ISSUE_TEMPLATE/new_meetup.md

```diff
@@ -0,0 +1,26 @@
+***Please help us by filling out the text below. All answers are optional***
+
+# New Meetup
+
+## Where
+
+*A short description of the event. Please also add any links, e.g. to the meetup page.*
+
+## When
+
+*the date. please name the timezone*
+
+[ ] I'd like the event to be public calendar
+[ ] It's a repeat event, I would like to have access to the calender to do things on my own
+
+## Support
+
+[ ] I need coaching
+[ ] I need contacts in my area
```

Comment:
this sounds like a dating website copy - try something like `I would like to be introduced to fellow meetup organizers in my area`